tests/mashupDocumentParser.test.ts

```diff
@@ -0,0 +1,36 @@
+import MashupHandler from "../src/mashupDocumentParser";
+import { arrayUtils, pqUtils } from "../src/utils";
+import { simpleQuery, section1mMock } from "./mocks";
+import base64 from "base64-js";
+import JSZip from "jszip";
+import WorkbookTemplate from "../src/workbookTemplate";
+import { section1mPath } from "../src/constants";
+describe("Mashup Document Parser tests", () => {
+    test("ReplaceSingleQuery test", async () => {
+        const mashupHandler = new MashupHandler();
+
+        const defaultZipFile = await JSZip.loadAsync(
+            WorkbookTemplate.SIMPLE_QUERY_WORKBOOK_TEMPLATE,
+            { base64: true }
+        );
+        const base64str = await pqUtils.getBase64(defaultZipFile);
+
+        const replacedQueryStr = await mashupHandler.ReplaceSingleQuery(
+            base64str,
+            simpleQuery
+        );
+
+        const buffer = base64.toByteArray(replacedQueryStr).buffer;
+        const mashupArray = new arrayUtils.ArrayReader(buffer);
+        const startArray = mashupArray.getBytes(4);
+        const packageSize = mashupArray.getInt32();
+        const packageOPC = mashupArray.getBytes(packageSize);
+
+        const zip = await JSZip.loadAsync(packageOPC);
+        const section1m = await zip.file(section1mPath)?.async("text");
```

Comment:
This looks like a good candidate for a util function - getting the section1m from the base64 string, it would likely be useful in the future.

Reply:
it was the same as `getSection1m` in the previous PR, I returned it to the `mashupHandler` class. maybe it's good to have it separated from the class as a util, but inside the same file, or `mashupUpUtils` file.

Reply:
Yes with the tests it seems we have 'product' logic in test code, so that might tip the scale a bit towards having this part of the product code.
Not a must fix for now, resolving.